src/05-excitation-transfer.md

```diff
@@ -0,0 +1,762 @@
+```
+
+How intriguing, the energy of level 3 is exactly 1.5 - just as as it was when we didn't have any coupling between the TSS and the boson field.
+
+Now let's look at the corresponding eigenstates using a helper function `prettify_states` that we imported from the helper file.
+
+```python
+prettify_states(ekets, nmm_list)[[2,3]]
+```
+
+We can see that the eigenstates corresponding to levels 2 and 3 are normalised versions of: 
+- Level 2 = $|1,+,-\rangle  + |1,-,+\rangle$ +  other states with boson number different from 1
+- Level 3 = $|1,+,-\rangle  - |1,-,+\rangle$ 
+
+This pattern persists for all $\Delta E$ - try it out for yourself by changing the $\Delta E$ in the above Hamiltonian. There is always a horizontal level corresponding to  $|1,+,-\rangle  - |1,-,+\rangle$ with energy 1.5. This level (and others like it with different boson number) does not couple to others and so does indeed live in its own sub-universe.
+
+Why is it that changing plus to minus in $|1,+,-\rangle  \pm |1,-,+\rangle$ makes such a difference? What is the nature of this other sub-universe? They are important questions but they take us somewhat away from the primary mission of this tutorial. We'll come back to this in the next tutorial.
+
+> For those who just can't wait that long, it has to do with conservation of spin angular momentum
+
+For now, what we can confidently say is that when we see a 4 level anti-crossing it's actually only 3 levels that are interacting to produce the energy splitting - the other level is a non-interacting one of the form $|n,+,-\rangle  - |n,-,+\rangle$. With that said, let's take a look at these unfamiliar many level anti-crossings.
+
+
+### 5.6.3 - Beyond simple anti-crossings
+
+
+In Fig 7, there is a 4 level "anti-crossing" around $\Delta E = 1$. Because $\Delta E = \omega$ at this point, we suspect that the transition $|0,+, + \rangle \rightarrow |2,-, - \rangle$ (that we spoke of during exploration of the Hinton diagram) might be related to it.
+
+Let's zoom in a take a closer look before we head straight into simulation.
+
+```python
+df_even = make_df_for_energy_scan("$\Delta E$", 0.7, 1.3, 201, two_state_1.shape[0])
+
+for i, row in df_even.iterrows():
+    H =  row["$\Delta E$"]*two_state_1+ row["$\Delta E$"]*two_state_2 + 1*bosons + 0.1*interaction_1 + 0.1*interaction_2
+    evals, ekets = H.eigenstates()
+    df_even.iloc[i,1:] = evals 
+```
+
+```python
+df_even.plot(x="$\Delta E$",ylim=[1,2],legend=True, 
+        title=f"Even energy levels for {H_latex}   ($\omega=1$, $U=0.1$)    (Fig 9)",
+             figsize=(10,8));
+
+plt.ylabel("Energy");
+```
+
+When looking at levels 2 and 3 in Fig 9, we should try and ignore the horizontal part (the non interacting level) and instead imagine the elongated "S" looking shape as a single energy level - let's call it level S.
+
+We can see from Fig 9 that the notion of an anti-crossing is now somewhat ill defined because things don't look very symmetric. In particular:
```

Comment:
a side note on the legend of these plots: instead of level_X we might use Eigenstate_X or better Dynamic_Eigenstate_X although it is long. I don't remember whether we explicitly talk about this notation

Not to do it now, but let's think about it